### PR TITLE
Initial set of user docs translated xliff files from Crowdin

### PR DESCRIPTION
**Pull request by michaelDCurran**
This pr adds a newly generated changes.xliff for English, which has also been uploaded to Crowdin.

This PR updates the user docs github action to upload the English changes.xliff to Crowdin if it has changed.

This pr also includes the initial set of translated user docs xliff files from Crowdin.
So far that is 20 translations of the user guide, and 7 translations of changes (what's new).

scons will see that these are newer than their markdown files, and rebuild the markdown files from these, and then build the html from the rebuilt markdown files.

### CODE REVIEW

**Review by seanbudd — APPROVED** [review]

**Comment by michaelDCurran** [comment]
Err, this should be for beta.

**Comment by michaelDCurran** [comment]
Fixed now.

**Comment by zstanecic** [comment]
Hi,

As one of the translators, let's give my one penny to this discussion 
*smiles*

Can we remove changes for developers altogether from the changes file?

In the past, i haven't translated it for my languages, because developer 
changes should be just for developers, and developers know english to be 
able to read and have the access to the developer changes.

So, i always croslinked to english.

File: user_docs/en/changes.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # What's New in NVDA
 
 ## 2024.4
+Crowdin test string 1
```
[the remark this answers is not in the document]
Comment (michaelDCurran) [reply user_docs/en/changes.md]: Yes, It has been removed now. Again was necessary to test updating / uploading changes.xliff. Which is all working as expected.

**Comment by michaelDCurran** [comment]
@zstanecic  For now at least, we are going to keep the changes file as is. However, now that it is on Crowdin, translators can choose to simply not translate those strings if they wish.
We may revisit the structure of the changes file at some point, but only after we have completed the move to Crowdin.
I will have more to say on the translators list once this pr is merged.

**Comment by wmhn1872265132** [comment]
Please note that the .xliff file is now included into the installer, should this file be excluded?

**Comment by michaelDCurran** [comment]
@wmhn1872265132  thanks for catching this. I've excluded xliff files now.

**Review by seanbudd — APPROVED** [review]

**Comment by cary-rowen** [comment]
Hi @michaelDCurran 

Just wanted to ask:
Will we still have a way to directly edit the markdown version of the user guide in the future?

Since we plan to make extensive revisions to the Simplified Chinese version of the User Guide in the future, I would like to see a way to directly edit the Markdown version of the User Guide instead of a po file.

For large-scale changes, we may prefer to use a text editor to edit the markdown file.

Also want a script that uses structure comparison to ensure that the structure is not broken.

Thanks

**Comment by michaelDCurran** [comment]
@cary-rowen I'm sorry, but going forward  documentation such as the user guide and changes files can only be translated on Crowdin, either through its interface, or using poedit to translate the xliff files. 
If there are specific sections required by translations but is not yet in the English source, we can consider adding a placeholder in the English version.
I will have more to say on the translators list in the coming days once I have merged this pr.

**Comment by cary-rowen** [comment]
Thanks Mic,

Will NV Access open source the script for converting markdown in the future? This allows us to edit markdown to xliff in a highly customizable way

Best, 
Cary

**Comment by hwf1324** [comment]
Or can I use a project like https://github.com/cataria-rocks/md2xliff to do so?

**Pull request merged**